civihr_default_theme/includes/form.inc

```diff
@@ -3,3 +3,37 @@
  * @file
  * Theme and preprocess functions for forms
  */
+
+
+/**
+ * Implements theme_button().
+ */
+function civihr_default_theme_button($variables) {
```

Comment:
i think i need you to annotate here which bits are from radix and which bits you added yourself, cause i don't want to spend time reviewing code that comes straight from radix